install/kubernetes/cilium/values.yaml

```diff
@@ -579,6 +579,9 @@ encryption:
   # This option is only effective when encryption.type is set to ipsec.
   interface: ""
 
+# -- Configure the CIDR for the WireGuard Pod2Pod strict mode.
+strictModeCIDR: ""
```

Comment:
Let's move this into the above `encryption` section. This change also needs to be added to the `values.yaml.tmpl`.